doc/Makefile

```
@@ -10,15 +10,19 @@ SPHINXPROJ    = pyvista
 SOURCEDIR     = .
 BUILDDIR      = _build
 
+
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
 .PHONY: help Makefile
 
 .PHONY: all
+
+# allow experimental kill display
 all:
     echo PYVISTA_OFF_SCREEN: $$PYVISTA_OFF_SCREEN
+    export PYVISTA_KILL_DISPLAY = True
```

This has bugged me for a while: I'm pretty sure makefiles need tabs here, not spaces. My vim screams in red on this line, and `make all` is a no-op. And I don't think bash allows whitespace around `=` in definitions and exports, and I'm not sure why the same even works in the other Makefile.
```suggestion
	echo PYVISTA_OFF_SCREEN: $$PYVISTA_OFF_SCREEN
	export PYVISTA_KILL_DISPLAY=True
```
Most importantly, `all` doesn't actually compile anything, which seems like a mistake. Should it look something like this instead?
```makefile
all:
	echo PYVISTA_OFF_SCREEN: $$PYVISTA_OFF_SCREEN
	export PYVISTA_KILL_DISPLAY=True
	make phtml
```
?

---

This is something we should fix in a separate PR as I'm considering (again) dropping this (again).

---

OK, I'll go back to this bugging me for a while more until I get around to fixing it :laughing:

---

I'm fixing this in this PR, but I'm not finding that adding `phtml` had no effect.  I'm converting spaces to tabs.

---

So what does `make all` do for you right now?